### ISSUE TITLE
Add URL for reporting issues.

### ISSUE THREAD

**Issue by vpenades** [KhronosGroup/glTF-Project-Explorer#70]
Right now, for every project there's only one URL that points to the project's main site.

If possible, I would like the table to include these entries:

- An URL for issue reporting, if available.
- If the project is an exporter, the generator name prefix, ex: "babylon.js", "Sketchfab" , etc.

These two entries could be useful to add this functionality:

When an importer validates a glTF and finds errors, it can use the "generator" entry to find the project that produced the asset, and from there, report the Issue notification URL.

In other words, what I want to do is, when I load a glTF with errors, to be able to tell the user:

> The file xxxxx.gltf is malformed.
> Errors found: ....
> Warnings found ....
> Please, report these issues to http://www.somefancygltfexporter.com/issuereporting.html

**Comment by donmccurdy** [comment]
+1 for this suggestion — ideally we should also include the generator string for exporters, perhaps as a glob like `"FBX2GLTF *"`. This would enable viewers to look up the exporter for a given glTF file and match it to the bug URL. My viewer does this now, although I haven't done a great job of pushing the feature:

![Screen Shot 2020-10-22 at 9 57 39 AM](https://user-images.githubusercontent.com/1848368/96905217-0858b280-144d-11eb-8c1f-3f6055c1fa6e.png)

We could probably just merge https://github.com/KhronosGroup/glTF-Generator-Registry into this project, if we want to go down that path. There's not much incentive for exporters to add themselves to the generator registry, whereas there is actual incentives to register here, so I expect we'll get much more traction.

**Comment by weegeekps** [comment]
I like this idea. Give me a few days and I will figure out how best for us to handle this.

To be clear, what I'm looking at is adding:

* A field that projects can use for providing a link for issue reporting.
* A field that specifies the generator name prefix.

I'm also open to us merging the data from https://github.com/KhronosGroup/glTF-Generator-Registry as well, but it might be worthwhile to have a discussion about what fields we need. Perhaps, a topic for next weeks call?

**Comment by vpenades** [comment]
If possible, I would like the list to handle bin public projects; for example, sketchfab might probably don't have an entry, but I would have it to be included, so bugs can be reported to their forums
